src/backend/db/products.js

```diff
@@ -7,24 +7,182 @@ import { v4 as uuid } from "uuid";
 
 export const products = [
   {
-    _id: uuid(),
-    title: "You Can WIN",
-    author: "Shiv Khera",
-    price: "5000",
-    categoryName: "non-fiction",
-  },
-  {
-    _id: uuid(),
-    title: "You are Winner",
-    author: "Junaid Qureshi",
-    price: "3000",
-    categoryName: "horror",
-  },
-  {
-    _id: uuid(),
-    title: "Think and Grow Rich",
-    author: "Shiv Khera",
-    price: "1000",
-    categoryName: "fiction",
-  },
+    id:"a1",
```

Comment:
maintain  id as  _id:uuid(),  because  if you maintain static id, it causes a lot of problems later while implementing  cart and wishlist functionality and becomes  hard to debug

Reply:
I have made the changes 
I thought that uuid changes on every refresh so that might cause a problem if the page is refreshed